assets/ExpressionMetadata.json

```
@@ -99,6 +99,13 @@
             "minimumArguments": 1,
             "maximumArguments": 1
         },
+        {
+            "name": "dateTimeAdd",
+            "expectedUsage": "dateTimeAdd(base, duration, [format])",
+            "description": "Adds a time duration to a base datetime value.",
```

The type is actually string and not datetime
```suggestion
            "description": "Adds a time duration to a base value. ISO 8601 format is expected.",
```

---

@alex-frankel can you advise? I've taken the description verbatim from docs - thanks.

https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-date#datetimeadd

---

Yeah I guess datetime could be read as a type, when the intention was a format for the string. This looks good. @tfitzmac FYI - we should probably update the doc as well

---

Thanks, I'll update the doc today.